WordPress/Classes/Networking/ReaderTopicServiceRemote.m

```diff
@@ -174,9 +168,24 @@ - (NSString *)sanitizeTopicNameForAPI:(NSString *)topicName
         topicName = [topicName stringByReplacingOccurrencesOfString:@"--" withString:@"-"];
     }
 
+    topicName = [topicName stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
+
     return topicName;
 }
 
+- (NSArray *)normalizeMenuTopicsList:(NSArray *)rawTopics subscribed:(BOOL)subscribed recommended:(BOOL)recommended
+{
+    NSMutableArray *topics = [NSMutableArray array];
+    for (NSDictionary *topicDict in rawTopics) {
+        // Failsafe
+        if (![topicDict isKindOfClass:[NSDictionary class]]) {
```

Comment:
I think its pretty safe. We control the endpoint and would coordinate any changes with an API version bump.  Tho, fwiw I want to change many things about how we retrieve and parse tag topics so this will probably change in the near future anyway.